docs/src/test-advanced-js.md

```diff
@@ -120,7 +120,7 @@ In addition to everything from the [`workerInfo`](#workerinfo), the following in
 - `timeout: number` - Test timeout.
 - `annotations` - [Annotations](./test-annotations.md) that were added to the test.
 - `snapshotSuffix: string` - Suffix used to locate snapshots for the test.
-- `snapshotPath(snapshotName: string)` - Function that returns the full path to a particular snapshot for the test.
+- `snapshotPath(...pathSegments: string[])` - Function that returns the full path to a particular snapshot for the test.
```

Comment:
This seems to control argument types for `TestInfo` inside `types/test.d.ts` but for some reason it keeps overriding the type for `snapshotPath` in `types/test.s.ts` to `snapshotPath: (pathSegments: string[]) => string;`. Any ideas?

Reply:
Ugh nvm it was `utils/generate_types/overrides-test.d.ts` ac578b1